config.yaml

```
@@ -32,7 +32,7 @@ Standards:
         Path: /00-study-plan/study-plan.checkpoint.md
   -
     Title: Algorithmic Approaches
-    Description: Introduction to Algorithms, Analysis of Algorithms, Divide and Conquer, Recursion, Dynamic Programming
+    Description: Introduction to Algorithms, Analysis of Algorithms, Divide and Conquer, Recursion, Dynamic Programming (Optional)
```

How about a description like this? This keeps it to the non-optional topics (otherwise why does DP get listed but not sliding window) and fixes the ordering.
```suggestion
    Description: Introduction to Algorithms, Analysis of Algorithms, Recursion Review, Divide and Conquer, ...
```